belgif-rest-problem/src/test/java/io/github/belgif/rest/problem/ExpiredAccessTokenProblemTest.java

```diff
@@ -18,6 +19,8 @@ void construct() {
         assertThat(problem.getStatus()).isEqualTo(401);
         assertThat(problem.getDetail())
                 .isEqualTo("The Bearer access token found in the Authorization HTTP header has expired");
+        assertThat(problem.getHttpResponseHeaders()).containsEntry(HttpResponseHeaders.WWW_AUTHENTICATE,
```

Comment:
Shouldn't we also validate the header is properly set in our integration tests?

Reply:
I'm not sure if this is really needed, as we already have an integration test for the Retry-After HTTP header, which uses the same .getHttpResponseHeaders() mechanism?

Reply:
Maybe not, I think it's personal preference. I usually treat the application as a "black box" testing all scenarios (not considering shared logic) but as you say it's tested through Retry-After.